tests/test_utils.py

```diff
@@ -258,6 +263,30 @@ class MockException(Exception):
         with self.assertRaises(MockException):
             utils.update_available(silent_fail=False)
 
+    @patch('shub.utils._is_deploy_successful', return_value=True)
+    def test_echo_short_log_if_deployed_true(self, mock_dep_success):
+        log_file = LogFile()
+        last_logs = ["last log line"]
+        verbose = True
+        utils.echo_short_log_if_deployed(last_logs, log_file, verbose)
+        self.assertEqual(None, log_file.delete)
+
+        verbose = False
+        utils.echo_short_log_if_deployed(last_logs, log_file, verbose)
+        self.assertEqual(None, log_file.delete)
+
+    @patch('shub.utils._is_deploy_successful', return_value=None)
+    def test_echo_short_log_if_deployed_none(self, mock_dep_success):
```

Comment:
These two tests have very similar overhead. Maybe there's a way to unify them? (You can change the return value during the test by setting `mock_dep_sucess.return_value`)